packages/rancher-cis-benchmark/charts/templates/configmap.yaml

```diff
@@ -6,6 +6,8 @@ metadata:
 data:
   # Default ClusterScanProfiles per cluster provider type
   rke: "rke-profile-permissive"
+  rke2: "rke2-profile-hardened"
+  rke2: "rke2-profile-permissive"
```

Comment:
just add one profile to this configmap - its the profile to be run by default if user does not selects one